yarn-project/noir-compiler/src/contract-interface-gen/programTypescript.ts

```diff
@@ -192,30 +185,3 @@ function generateTsInterface(abiObj: NoirFunctionAbi): string {
     result
   );
 }
-
-const circuits = [
-  'private_kernel_init',
-  'private_kernel_inner',
-  'private_kernel_ordering',
-  'public_kernel_private_previous',
-  'public_kernel_public_previous',
-  'rollup_base',
-  'rollup_merge',
-  'rollup_root',
-];
-
-const main = async () => {
-  for (const circuit of circuits) {
-    const rawData = await fs.readFile(`./src/target/${circuit}.json`, 'utf-8');
-    const abiObj: NoirCompiledCircuit = JSON.parse(rawData);
-    const generatedInterface = generateTsInterface(abiObj.abi);
-    await fs.writeFile(`./src/types/${circuit}_types.ts`, generatedInterface);
-  }
-};
-
-try {
-  await main();
-} catch (err: unknown) {
-  log(`Error generating types ${err}`);
-  process.exit(1);
-}
```

Comment:
ah yes it was moved that way, but we have not moved to noir-wasm for compiling protocol circuits (will see how compile times change, may stick with nargo if CI slows down).

so the "loop" and call logic remains in noir-protocol-circuits, so that the types are generated regardless of if we use noir-wasm or nargo (and we can remove it, if we switch to noir-wasm)